.github/workflows/ci.yaml

```
@@ -10,10 +10,10 @@ jobs:
       strategy:
         matrix:
           os: [macos-latest, ubuntu-22.04, windows-latest]
-          python: ['3.7', '3.8', '3.9', '3.10']
+          python: ['3.8', '3.9', '3.10', '3.11']
           include:
           - os: ubuntu-20.04
-            python: '3.6'
+            python: '3.8'
```

We should keep 3.6 as long as we need to support RHEL 8 (and as long as `actions/setup-python` will allow us to).

```suggestion
            python: '3.6'
```